apps/web/src/views/Swap/components/HotTokenList/SwapTokenTable.tsx

```diff
@@ -302,7 +306,7 @@ const TokenTable: React.FC<
     return <Skeleton />
   }
   return (
-    <TableWrapper>
+    <TableWrapper style={{ maxHeight: 450, overflow: 'auto' }}>
```

Comment:
We should only scroll within the table, table header shouldn't be part of it. Also let's just add scrollbar on mobile device

Reply:
fixed